images/build/Dockerfiles/vso.Dockerfile

```diff
@@ -6,6 +6,30 @@ ARG GIT_COMMIT=unspecified
 ARG BUILD_NUMBER=unspecified
 ARG RELEASE_TAG_NAME=unspecified
 
+COPY build/__condaConstants.sh /tmp/__condaConstants.sh
+RUN apt-get update \
+    && apt-get install -y --no-install-recommends \
+        apt-transport-https \
+    && rm -rf /var/lib/apt/lists/*
+RUN curl https://repo.anaconda.com/pkgs/misc/gpgkeys/anaconda.asc | gpg --dearmor > conda.gpg
+RUN install -o root -g root -m 644 conda.gpg /usr/share/keyrings/conda-archive-keyring.gpg
+RUN gpg --keyring /usr/share/keyrings/conda-archive-keyring.gpg --no-default-keyring --fingerprint 34161F5BF5EB1D4BFBBB8F0A8AEB4F8B29D82806
+RUN echo "deb [arch=amd64 signed-by=/usr/share/keyrings/conda-archive-keyring.gpg] https://repo.anaconda.com/pkgs/misc/debrepo/conda stable main" > /etc/apt/sources.list.d/conda.list
+RUN . /tmp/__condaConstants.sh \
+    && apt-get update \
+    && apt-get install -y --no-install-recommends \
+        conda=${CONDA_VERSION} \
+    && rm -rf /var/lib/apt/lists/*
+ENV CONDA_SCRIPT="/opt/conda/etc/profile.d/conda.sh"
+RUN . $CONDA_SCRIPT && \
```

Comment:
@kichalla .. can we please consolidate all these run statements into one or two run statements by using bash script or something? so from line 10 to line 22

Reply:
Sure, will do.